pass.go

```diff
@@ -3,10 +3,28 @@ package gopass
 import (
 	"errors"
 	"fmt"
+	"io"
+	"os"
+
+	"golang.org/x/crypto/ssh/terminal"
 )
 
+var defaultGetCh = func() (byte, error) {
```

Comment:
Added this as a separate variable to further aid testing; that way you never lose a reference to the default function.